build-tools-internal/src/main/groovy/elasticsearch.local-distribution.gradle

```diff
@@ -29,6 +30,6 @@ tasks.register('localDistro', Sync) {
   from(elasticsearch_distributions.local)
   into("build/distribution/local")
   doLast {
-    logger.lifecycle("Elasticsearch distribution installed to ${destinationDir}.")
+    logger.lifecycle("Elasticsearch distribution installed to ${destinationDir}/elasticsearch-${VersionProperties.elasticsearch}")
```

Comment:
I changed this because I was annoyed that I couldn't copy-paste the directory and end up in the unpacked distro.

Reply:
:+1: